code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -153,3 +153,8 @@
 			on = FALSE
 	else
 		return ..()
+
+/obj/machinery/portable_atmospherics/scrubber/CtrlShiftClick(mob/user)
+	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
```

Comment:
You don't want this to work for silicons from further away?
Ignore if intentional